src/GovUk.Education.ExploreEducationStatistics.Content.Api/Controllers/ThemeController.cs

```diff
@@ -46,5 +51,12 @@ public async Task<ActionResult<List<AllMethodologiesThemeViewModel>>> GetMethodo
                 .GetSummariesTree()
                 .HandleFailuresOrOk();
         }
+
+        [HttpGet("themes-list")]
```

Comment:
I wasn't sure what to name this request, given that `GetPublicationTree` already uses the route `/api/themes`. Suggestions appreciated! 🙂